Applications/Spire/Source/Ui/DropShadow.cpp

```diff
@@ -43,6 +43,9 @@ DropShadow::DropShadow(bool is_menu_shadow, bool has_top, QWidget* parent)
   setAttribute(Qt::WA_TranslucentBackground);
   setAttribute(Qt::WA_ShowWithoutActivating);
   m_parent->window()->installEventFilter(this);
+  connect(m_parent,
+    static_cast<void (QObject::*)(QObject*)>(&QObject::destroyed),
```

Comment:
Just to confirm, the following didn't work?

    connect(m_parent, &QObject::destroyed, ...);

Reply:
No, the compiler complained something to the effect of 'this doesn't evaluate to an expression taking one parameter'.

Reply:
Good, use ```qOverload``` from now on:

    connect(m_parent, qOverload<>(QObject::destroyed), ...);